EnglishTranslation-master/ETC.tsv

```diff
@@ -13642,11 +13642,11 @@ ETC_20150729_013643	Sanctum's Blessing - Increased Attack
 ETC_20150729_013644	Increases your attack due to the sanctum's blessing.
 ETC_20150729_013645	A strong protective shield is blocking all attacks.
 ETC_20150729_013646	Divine Power bestowed on Unknown Ruins
-ETC_20150729_013647	Imposes divine punishment on all the evil beings things near the ruins.
+ETC_20150729_013647	Imposes divine punishment on all evil beings beings near the ruins.
```

Comment:
"beings beings" ? 
Do those 'beings' belong to one 'evil being'? If so, then:
**Imposes divine punishment on all evil being's beings / Imposes divine punishment on all evil being's creatures (etc.)**
If not, then:
**Imposes divine punishment on all evil beings near the ruins.**